js/events.js

```diff
@@ -396,7 +396,7 @@ function handleElementDoubleClick(elementView) {
                       <hr>
                       <div class="form-group">
                         <label for="initialisation">Initialisation</label>
-                        <input type="text" class="form-control" id="initialisation"
+                        <input type="text" placeholder="<counter> = value/variable" class="form-control" id="initialisation"
```

Comment:
Since the other two are examples, let this also follow the same format.
Ex: counter = variable